modules/@apostrophecms/modal/ui/apos/mixins/AposDocsManagerMixin.js

```diff
@@ -99,6 +99,12 @@ export default {
     }
   },
   methods: {
+    preview(doc) {
+      window.open(doc._url, '_blank').focus();
+    },
+    findDoc(id, docs) {
```

Comment:
I would go with findDocById(docs, id). We generally put the haystack before the needle.